src/ray/raylet/worker_pool.cc

```diff
@@ -16,27 +16,46 @@ WorkerPool::WorkerPool(int num_workers, const std::vector<std::string> &worker_c
   // become zombies instead of dying gracefully.
   signal(SIGCHLD, SIG_IGN);
   for (int i = 0; i < num_workers; i++) {
-    StartWorker();
+    // Force-start num_workers workers.
+    StartWorker(true);
   }
 }
 
+/// A constructor that initializes an empty worker pool with zero workers.
+WorkerPool::WorkerPool(const std::vector<std::string> &worker_command)
+    : worker_command_(worker_command) {}
+
 WorkerPool::~WorkerPool() {
   // Kill all registered workers. NOTE(swang): This assumes that the registered
   // workers were started by the pool.
+  // TODO(atumanov): remove killed workers from the pool.
```

Comment:
Does this TODO mean we should also kill all the PIDs in `started_worker_pids_`? Either way, we should probably do that here in this PR.